src/poset/poset_test.go

```diff
@@ -1407,63 +1427,181 @@ func TestInsertEventsWithBlockSignatures(t *testing.T) {
 }
 
 /*
-                   Round 8
-      [m0]  | [m2]-----------------------------
-		| \ | / |  Round 7
-		|  <l1> |
-		|  /|   |
-	  <l0>  |   |
-		| \ |   |
-		|   \   |
-		|   | \ |
-		|   | <l2>-----------------------------
-		|   | / |  Round 6
-		| [k10] |
-		| / |   |
-	  [k0]  | [k2]-----------------------------
-		| \ | / |  Round 5
-		| <j1>  |
-		|  /|   |
-	  <j0>  |   |
-		| \ |   |
-		|   \   |
-		|   | \ |
-	    |   | <j2>-----------------------------
-		|   | / |  Round 4
-		| [i1]  |
-		| / |   |
-	  [i0]  | [i2]-----------------------------
-		| \ | / |  Round 3
-		| <h10> |
-		|  /|   |
-	   h0b  |   |
-		|   |   |
-	  <h0>  |   |
-		| \ |   |
-		|   \   |
-		|   | \ |
-	---g0x  | <h2>----------------------------- //g0x's other-parent is f2. This situation can happen with concurrency.
-	|	|   | / |  Round 2
-	|	|  g10  |
-	|	| / |   |
-	|  [g0] | [g2]
-	|	| \ | / |
-	|	| [g1]  | ------------------------------
-	|	|   |   |  Round 1
-	|	| <f1>  |
-	|  	|  /|   |
-	| <f0>  |   |
-	|	| \ |   |
-	|	|   \   |
-	|	|   | \ |
-	|   |   |  f2b
-	|	|   |   |
-	----------<f2>------------------------------
-		|   | / |  Round 0
-		|  e10  |
-	    | / |   |
-	   [e0][e1][e2]
-		0   1    2
+                 +-----------------------------------------------------+
```

Comment:
In my opinion, previous ascii-grafics are more readable than new.

Reply:
@devintegral Yeah, so I didn't know you were working on it also. I asked @dev10 to work on this earlier, and we were sharing code + output snippets in Slack. Got Peter to open this thread for Maxim to review.

We will need bidirectional generation, from ASCII graphic to graphviz and back again, in addition to going from the graph in Go to a graphviz DOT, then back from DOT to Go.

Reply:
@SamuelMarks, how do you propose to use translation from graphviz to ASCII graphic? I can't imagine real case you are about.